packages/fast-sketch-library/src/aSketchPage.ts

```diff
@@ -0,0 +1,51 @@
+import { Base, nodeToSketchLayers, Page, SymbolMaster, Text,  } from "@brainly/html-sketchapp";
+import { ISymbolLibrarySource } from "./index";
+const symbolNameDataAttribute: string = "data-sketch-symbol";
+
+export function getAsketchSymbols(source: ISymbolLibrarySource): JSON[] {
+    const selectors: string = Array.isArray(source.selectors) ? source.selectors.join(", ") : source.selectors;
+    const nodes: Element[] = Array.from(document.querySelectorAll(selectors));
+
+    return nodes.map((node: Element) => {
+        const rect: ClientRect = node.getBoundingClientRect();
```

Comment:
Generic comment in a specific place. In our implementation we expect the DOM, but we can't count on that for others. Should these functions ensure that the DOM is available, or should we note that this utility expects access to the DOM, so those implementing it should be aware of that?

Reply:
I can make a note - yea this is 100% only executable in the dom, and honestly should error if the DOM isn't available so it doesn't fail silently.

Reply:
Sounds good - just figure at minimum we should note that it's a requirement due to the internals.